pagopa/io-dev-api-server
# [IOCOM-208] Migrate to ESLint and restore Prettier

This PR depends on #246

## Short description
This PR migrates from TSLint (which was not working anymore) to ESLint
It also updates Prettier to its latest version and restores/adds it to the CI

## List of changes proposed in this pull request
There are a lot of changes in this PR. They have been done to support ESLint, remove TSLint and fix all lint errors.
Lint warnings have been left for future developments to solve

## How to test
Run `yarn prettify` and `yarn prettier:check` to verify that prettier is working
Run `yarn lint` and check that it has an output with warnings but no errors
